dist/profile/manifests/ldap.pp

```diff
@@ -241,7 +241,12 @@
     action => 'accept',
   }
 
-
+  firewall { '107 accept inbound LDAPS from nginx.azure.jenkins.io':
+    proto  => 'tcp',
+    source => 'nginx.azure.jenkins.io.',
```

Comment:
Indeed it will be the source IP for outbound packets.
But as implementation details 'often' change in k8s, it's a good idea to use beta.accounts instead of nginx.